FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Ankit on 30/07/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+final class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var feed_description: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedCache
```

Comment:
`cache` is marked as optional in `FeedStore.xcdatamodel`. Please make it optional here so that the class and the model are consistent

```suggestion
	@NSManaged var cache: ManagedCache?
```